app/controllers/screenings_controller.rb

```diff
@@ -14,7 +14,9 @@ def create
                   date: params[:screening][:date], 
                   movie_id: params[:movie_id],
                   cinema_hall_id: params[:cinema_hall_id] })
-    
+      #doesn't work
```

Comment:
Maybe place `byebug` inside of this method and check how your params look like :)

Reply:
Look also on the `screening_params` method and what params are permited.

Reply:
I added `:movie_id `and `:cinema_hall_id` but it didn't change anything 
```
   def screening_params
      params.require(:screening).permit(:date, :movie_id, :cinema_hall_id)
    end
```
   When I run byebug and check `screening_params` there are not there. So I guess I have to use `params` separately like I did?
   
<img width="882" alt="montecinema — fsevent_watch ◂ localhost:3000)  montecinema  RBENV_VERSION=3 0 1 TERM_PROGRAM=Apple_Terminal — 126×47 2021-05-28 13-59-14" src="https://user-images.githubusercontent.com/42643953/119981363-d51b6a80-bfbd-11eb-81e3-0cce7640c430.png">

Reply:
Your `screening` parameters should all be under the `screening` key. If we take the above screenshot as an example it should look like this:
```
{ "screening" => { "date" => "2021-05-28", "cinema_hall_id" => "1", "movie_id" => "12" }, "controller" => "screening", "action" => "create" }
```

Reply:
Ok. Any idea why there are not? Where can I start looking for a bug?